src/pages/groupComparison/pathwayMapper/GroupComparisonPathwayMapper.tsx

```diff
@@ -140,6 +150,11 @@ export default class GroupComparisonPathwayMapper extends React.Component<
                                 patientView={false}
                                 groupComparisonView={true}
                                 activeGroups={this.props.activeGroups as any}
+                                currentPathway={this.props.currentPathway}
+                                rankingChoices={this.props.rankingChoices}
+                                updateRankingChoices={
+                                    this.props.updateRankingChoices
+                                }
```

Comment:
Let's refer to a `userSelectionStore` instead.

```suggestion
                                currentPathway={this.userSelectionStore.currentPathway}
                                rankingChoices={this.userSelectionStore.props.rankingChoices}
                                updateRankingChoices={
                                    this.userSelectionStore.updateRankingChoices
                                }
```